Add breachedPasswordStatus to response examples and api doc (#4243)
FusionAuth/fusionauth-site: astro/src/content/docs/apis/_user-registration-combined-response-body.mdx

    The User's birthdate formatted as `YYYY-MM-DD`
  </APIField>
  <APIField name="user.breachedPasswordLastCheckedInstant" type="Long">
    The [instant](/docs/reference/data-types#instants) this user's password was last checked for inclusion in the set of known breached credentials.
  </APIField>
  <APIField name="user.breachedPasswordStatus" type="String">

>     If a user has been compromised, this field shows the level of compromise from the following options:

Reviewer: ```suggestion
   This field shows the type of compromise based the following options:
```

The existence of `none` implies we send this even when no compromise has occurred.

Author: ```suggestion
    Shows whether or not a user's password has been compromised, based on a comparison with a list of known breached passwords. May have any of the following values:
```
Good point, I wouldn't want folks to merely check for the presence of this field to determine the breach status. Hopefully this is an improvement!
